GoInfoGame/GoInfoGame/quests/StairNumber/StairNumberForm.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  StairNumberForm.swift
+//  GoInfoGame
+//
+//  Created by Lakshmi Shweta Pochiraju on 18/01/24.
+//
+
+import Foundation
+import SwiftUI
+
+struct StairNumberForm: View, QuestForm {
+    func applyAnswer(answer: Int) {
+    }
+    typealias AnswerClass = Int
+    @State private var numberOfSteps: Int = 0
+    @State private var showAlert = false
+    @State private var isInputValid: Bool = false
+    var onConfirm: ((_ numberOfSteps: Int) -> Void)?
+    
```

Comment:
write a different function that handles the response and does `applyAnswer`

Reply:
Resolved.